BookingSystem/BookingDatabase/Script.PostDeployment.sql

```diff
@@ -0,0 +1,163 @@
+﻿/*
+Post-Deployment Script Template							
+--------------------------------------------------------------------------------------
+ This file contains SQL statements that will be appended to the build script.		
+ Use SQLCMD syntax to include a file in the post-deployment script.			
+ Example:      :r .\myfile.sql								
+ Use SQLCMD syntax to reference a variable in the post-deployment script.		
+ Example:      :setvar TableName MyTable							
+               SELECT * FROM [$(TableName)]					
+--------------------------------------------------------------------------------------
+*/
+
+
+IF (EXISTS(SELECT * FROM [dbo].[Users]))
+BEGIN
+    DELETE FROM [dbo].[Users]
```

Comment:
This is a very radical decision=) Imagine, you have thousands of data in your Prod database. Users will start a "new life" after redeployment. If you want to insert data only once after database is created, you should check if there are any data in a table and if no insert them